src/region/RegionCreation.tsx

```diff
@@ -55,24 +55,21 @@ export default class RegionCreation extends React.PureComponent<Props, State> {
 
     render(): JSX.Element | null {
         const { isCreating, isDiscoverabilityEnabled, isRotated, staged } = this.props;
-        const canCreate = isCreating && !isRotated;
-
-        if (!canCreate) {
```

Comment:
Should this be changed to `isRotated` instead of being removed? Otherwise have to check it twice below.